Code-4-Community/speak-for-the-trees-frontend
# Optimize login/signup for mobile

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/925293934/pulses/925294541)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

Makes the log in and signup pages easier to navigate on mobile devices/narrow windows

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.
- Updated to use styled components only
- In the Login and Signup components, only displays the form (no input or greeting container) if mobile
  - Created a SignupForm and LoginForm component, form layouts are dependent on whether the browser is mobile (rather than making mobile page components so the login/signup form only exists in one place)

## Screenshots

Provide screenshots of any new components, styling changes, or pages.
Login
<img width="630" alt="LoginDesktop" src="https://user-images.githubusercontent.com/22990100/103046048-c6cfab00-4554-11eb-8438-c148c3768b0a.png"> <img height="375" alt="LoginMobile" src="https://user-images.githubusercontent.com/22990100/103046050-c8996e80-4554-11eb-9ba7-c404db07d70a.png"> 

Signup
<img width="630" alt="SignupDesktop" src="https://user-images.githubusercontent.com/22990100/103046059-cd5e2280-4554-11eb-9ee4-b135da21f73d.png"> <img height="375" alt="SignupMobile" src="https://user-images.githubusercontent.com/22990100/103046060-ce8f4f80-4554-11eb-813d-645fc3d22477.png">

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

Visually confirmed the layout updated when resizing the browser, used Device Mode in Chrome Dev Tools to make sure the layout worked for mobile.
